GitUI/Editor/FileViewer.cs

```diff
@@ -786,10 +786,13 @@ private void CopyNotStartingWith(char startChar)
                     code = " " + code;
                 }
 
-                var lines = code.LazySplit('\n')
-                    .Where(s => s.Length == 0 || s[0] != startChar || (s.Length > 2 && s[1] == s[0] && s[2] == s[0]));
                 var hpos = fileText.IndexOf("\n@@");
 
+                var lines = code.LazySplit('\n')
+                    .Where(s => !s.StartsWith("@@")
+                    && code.IndexOf(s) > hpos
+                    && (s.Length == 0 || s[0] != startChar || (s.Length > 2 && s[1] == s[0] && s[2] == s[0])));
```

Comment:
Please indent the lines to clearly indicate which part of the expression these two lines belong to